Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py

```diff
@@ -280,308 +104,260 @@ def parse_reputation_human_readable(reputation):
     except Exception as e:
-        demisto.info(f'McAfee failed to get file reputation with error: {str(e)}')
+        demisto.info("McAfee failed to get file reputation with error: " + str(e))
         return None
     return res
 
 
-def get_hash_type_key(file_hash: str):
-    hash_type = get_hash_type(file_hash)
-    hash_type_key = HASH_TYPE_KEYS.get(hash_type, None)
-    if not hash_type_key:
-        raise Exception(f'Invalid value, {file_hash} is not a valid SHA1, SHA256 or MD5 value.')
-    return hash_type_key
-
-
-def get_file_instance(dbot_score: Union[Common.DBotScore, None], file_hash: str, hash_type_key: str) -> Common.File:
-    return Common.File(dbot_score=dbot_score,
-                       md5=file_hash if hash_type_key == HashType.MD5 else None,
-                       sha1=file_hash if hash_type_key == HashType.SHA1 else None,
-                       sha256=file_hash if hash_type_key == HashType.SHA256 else None)
-
-
-def parse_reputation(provider_id: int, reputation: Dict):
-    file_reputation_parser = GeneralFileReputationParser.init(provider=provider_id)
-    parsed_reputation = file_reputation_parser.parse_data(reputation_data=reputation)
-    parsed_reputation['Provider_ID'] = provider_id
-    return parsed_reputation
-
-
-def parse_raw_result(raw_result: Dict, file_hash: str, reliability: str, hash_type_key: str):
-    context_data = {}
-    parsed_reputation_data = {}
-    max_trust_level = TrustLevel.KNOWN_TRUSTED_INSTALLER + 1  # More than the maximum possible trust level
-    lowest_trust_level = max_trust_level
-    reputations = raw_result.values()
-    for reputation in reputations:
-        # Parse the raw result of each reputation result
-        provider_id = reputation[FileReputationProp.PROVIDER_ID]
-        parsed_reputation_data[get_provider_abbr(provider_id=provider_id)] = parse_reputation(provider_id=provider_id,
-                                                                                              reputation=reputation)
-        # Get the vendor's data with the lowest score
-        rep_trust_level = reputation.get(FileReputationProp.TRUST_LEVEL, 0)
-        if rep_trust_level != 0 and rep_trust_level < lowest_trust_level:
-            lowest_trust_level = rep_trust_level
-    if lowest_trust_level == max_trust_level:
-        lowest_tl_score = {
-            LOWEST_TRUST_LEVEL_KEY: TrustLevel.NOT_SET,
-            LOWEST_SCORE_KEY: Common.DBotScore.NONE
-        }
-    else:
-        lowest_tl_score = {
-            LOWEST_TRUST_LEVEL_KEY: lowest_trust_level,
-            LOWEST_SCORE_KEY: trust_level_to_score(lowest_trust_level)
+def file(hash_inputs):
+    hash_list = []
+
+    for hash_value in hash_inputs:
+        config = get_client_config()
+        with DxlClient(config) as client:
+            client.connect()
+            # Create the McAfee Threat Intelligence Exchange (TIE) client
+            tie_client = TieClient(client)
```

Comment:
Is the configuration changed for each file? if not, no reason to do it inside the loop